samples/AWS/AWS.AppHost/AWS.AppHost.csproj

```diff
@@ -0,0 +1,17 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net8.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+    <IsAspireHost>true</IsAspireHost>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="$(RepoRoot)\src\Aspire.Hosting\Aspire.Hosting.csproj" />
+    <ProjectReference Include="$(RepoRoot)\src\Aspire.Hosting.AWS.DynamoDBLocal\Aspire.Hosting.AWS.DynamoDBLocal.csproj" />
```

Comment:
Please leave `ProjectReference` paths as VS (or VS Code) had put it.
Only use `$(RepoRoot)` in the MSBuild files (i.e., in *.props and *.targets)

Reply:
Done